encoding/protobuf/doc.go

```diff
@@ -54,11 +54,60 @@
 // are created for every RPC method: one that will handle the standard Protobuf
 // binary encoding, and one that will handle the JSON encoding.
 //
+// If coupled with an HTTP Inbound, Protobuf procedures can be called using
+// curl. Given the following Protobuf definition:
+//
+//   syntax = "proto3;
+//
+//   package foo.bar;
+//
+//   message EchoRequest {
+//     string value = 1;
+//   }
+//
+//   message EchoResponse {
+//     string value = 1;
+//   }
+//
+//   service Baz {
+//     rpc Echo(EchoRequest) returns (EchoResponse) {}
+//   }
+//
+// And the following configuation:
+//
+//   service:
+//     name: hello
+//   yarpc:
+//     inbounds:
+//       http:
+//          address: ":8080"
+//
+// If running locally, one could make the following call:
+//
+//   curl \
+//     http://0.0.0.0:8080 \
+//     -H 'context-ttl-ms: 2000' \
+//     -H 'rpc-caller: hello-client' \
+//     -H 'rpc-service: hello' \
+//     -H 'rpc-encoding: json' \
+//     -H 'rpc-procedure: foo.bar.Baz::Echo' \
+//     -d '{"value":"sample"}'
+//
+// Where context-ttl-ms is the timeout, rpc-caller is anything, rpc-service is the name
+// of the configured service, rpc-encoding is json, rpc-procedure is proto_package.proto_service::proto_method,
+// and the data is the JSON representation of the request.
+//
+// If using YAB, one can also use:
```

Comment:
nit: "yab" is never capitalized in our docs

Reply:
Fixed